tests/acceptance/ember-keyboard-test-auto-responder-test.js

```diff
@@ -0,0 +1,176 @@
+      await keyDown('ArrowRight');
+      assert.deepEqual(getValues(), [1, 1, 1], 'equal responders all respond');
+
+      await fillIn(
+        `${hook('counter-first')} ${hook('counter-priority-input')}`,
+        '1'
+      );
+      await blur(`${hook('counter-first')} ${hook('counter-priority-input')}`);
+
+      await keyDown('ArrowRight');
+      assert.deepEqual(
+        getValues(),
+        [2, 2, 2],
+        'highest responder responds first, lower responders follow'
+      );
+
+      await fillIn(
+        `${hook('counter-second')} ${hook('counter-priority-input')}`,
+        '1'
+      );
+      await blur(`${hook('counter-second')} ${hook('counter-priority-input')}`);
+      await click(
+        `${hook('counter-first')} ${hook(
+          'counter-stop-immediate-propagation-toggle'
+        )}`
+      );
+
+      await keyDown('ArrowRight');
+      assert.deepEqual(
+        getValues(),
+        [3, 2, 3],
+        'highest responder responds first and stops immediate propagation, lower responders follow'
+      );
+
+      await click(
+        `${hook('counter-first')} ${hook(
+          'counter-stop-immediate-propagation-toggle'
+        )}`
+      );
+      await click(
+        `${hook('counter-first')} ${hook('counter-stop-propagation-toggle')}`
+      );
+
+      await keyDown('ArrowRight');
+      assert.deepEqual(
+        getValues(),
+        [4, 3, 3],
+        'highest responders responds first and block propagation to lower priority responders'
+      );
+
+      await click(
+        `${hook('counter-first')} ${hook('counter-activated-toggle')}`
+      );
+
+      await keyDown('ArrowRight');
+      assert.deepEqual(
+        getValues(),
+        [4, 4, 4],
+        'deactivating a responder removes it from the stack, deactivated responders do not block propagation'
+      );
+
+      await fillIn(
+        `${hook('counter-first')} ${hook('counter-priority-input')}`,
+        '2'
+      );
+      await blur(`${hook('counter-first')} ${hook('counter-priority-input')}`);
+      await click(
+        `${hook('counter-first')} ${hook('counter-stop-propagation-toggle')}`
+      );
+      await click(
+        `${hook('counter-first')} ${hook('counter-activated-toggle')}`
+      );
+      await click(
+        `${hook('counter-first')} ${hook('counter-first-responder-toggle')}`
+      );
+      await click(
+        `${hook('counter-second')} ${hook('counter-first-responder-toggle')}`
+      );
+      await click(
+        `${hook('counter-second')} ${hook(
+          'counter-stop-immediate-propagation-toggle'
+        )}`
+      );
+
+      await click(
+        `${hook('counter-third')} ${hook('counter-first-responder-toggle')}`
+      );
+
+      await keyDown('ArrowRight');
+      assert.deepEqual(
+        getValues(),
+        [5, 5, 4],
+        'first responders get called in priority order.'
+      );
+    });
+
+    test('test event does not propagate on input field', async function (assert) {
```

Comment:
This is the key test of the file. Let's move it to the top and also add one that shows a key event firing associated with a non-input element and still triggering responders.

Reply:
moved that test to the top. What is wrong with the other tests In order to ensure that? Am I missing something? I copied all the other tests, because they test the standard features and added a test that makes sure input fields now do not propagate. Anything special you are looking for?